packages/devtools_app/integration_test/test/live_connection/performance_screen_event_recording_test.dart

```diff
@@ -27,10 +26,10 @@ void main() {
     logStatus(
       'Open the Performance screen and switch to the Timeline Events tab',
     );
-    await tester.tap(
-      find.widgetWithText(Tab, ScreenMetaData.performance.title),
-    );
-    await tester.pump(longPumpDuration);
+
+    await switchToScreen(tester, ScreenMetaData.performance);
+    await tester.pump(safePumpDuration);
```

Comment:
please switch this back to the longPumpDuration. IIRC, this test was flaky without it.

Reply:
switchToScreen already contains one pump. So I replaced one  longPumpDuration with two safePumpDuration, that should be enough.